src/lib/arrays.js

```diff
@@ -0,0 +1,95 @@
+const getNthElement = (index, array) => {
+  if (index >= array.length) {
+    return array[index - array.length];
+  }
+  return array[index];
```

Comment:
You can simplify this quite a bit using the modulus operator, which is similar to division except that instead of returning a decimal when dividing any remainder, it gives a whole number (EG 5 % 3 = 2). 
Since thats what you are in effect doing here when the index is greater than the length of the array you can rewrite this as follows:
```
  return array[index % array.length];
```